ui/tests/helpers/xterm.ts

```diff
@@ -0,0 +1,21 @@
+import { triggerEvent, triggerKeyEvent, focus } from '@ember/test-helpers';
+
+/**
+ * Returns the text from an xterm.js terminal element.
+ *
+ * Simulates select-all, then copy, then returns the resultant clipboard data.
+ *
+ * Assumes only one xterm instance is rendered.
+ */
+export async function getTerminalText(): Promise<string> {
+  focus('.xterm-helper-textarea');
```

Comment:
are we missing an `await` here?

Reply:
Yeah, I must have rush and committed an in-between state. Should be rectified now.